### DISCUSSION TITLE
MacOs Monterey 12.5

### DISCUSSION THREAD

**Discussion by BR00l** [asmvik/yabai#1335] [General]
Has anyone updated ?
Is it working as it should?

**Comment by asmvik** [comment]
Works fine yes.

**Comment by SichangHe** [comment]
Can confirm so far.

**Comment by jordanotsuji** [comment]
Do you have to keep SIP disabled for Monterey?

**Reply by BR00l** [reply]
No, if you dont need the extra functions.

**Comment by msr8** [comment]
Yupp, have been using it for a few days and loving it. The only issue I have had till now is a hotcorner issue when switching focus of windows, more detail at #1355
